### ISSUE TITLE
Convert multiple files from CLI using wildcards

### ISSUE THREAD

**Issue by dominic-simplan** [asciidoctor/asciidoctor-cli.js#102]
Hi,

is it possible to convert mulptile adoc files at once from the CLI using wildcards?
For example as described here: https://asciidoctor.org/docs/user-manual/#process-multiple-source-files-from-the-cli

However, this doesn't seem to work (at least on Windows) in asciidoctor.js:

```
PS> npx asciidoctor *.adoc  
ENOENT: no such file or directory, open '*.adoc'
PS> npx asciidoctor '*.adoc'
ENOENT: no such file or directory, open '*.adoc'
```

**Comment by Mogztter** [comment]
Hello @dominic-simplan 

> is it possible to convert mulptile adoc files at once from the CLI using wildcards?
For example as described here: https://asciidoctor.org/docs/user-manual/#process-multiple-source-files-from-the-cli

Yes it's possible but it's a shell feature. If supported, your shell will automatically expand the pattern. I'm not a Windows expert but apparently it does not work on Windows PowerShell :disappointed:

**Comment by dominic-simplan** [comment]
Hi,
thanks for the quick reply!
It seems like the defualt asciidoctor cli has a cross-platform way to deal with this:

> Since the globs in this command rely on shell expansion, the command is not portable across platforms. To make it portable, you can allow the Asciidoctor CLI to expand the globs. To do so, instruct the shell to not expand the glob by quoting the pattern, as shown here:
> 
> $ asciidoctor '*.adoc' '*/*.adoc'

So this is not supported in asciidoctor.js-cli I assume?

**Comment by Mogztter** [comment]
Indeed, I didn't know that the Asciidoctor (Ruby) CLI can expand globs.

> So this is not supported in asciidoctor.js-cli I assume?

No, currently it's not supported but since it's supported by Asciidoctor (Ruby) CLI we should also support it.
Can I rename this issue to make it clear that this issue is now about "expanding globs when the shell does not support it"?

**Comment by dominic-simplan** [comment]
Sure, and thanks for the help :-)

**Comment by Mogztter** [comment]
Here's the logic in Asciidoctor Ruby:

https://github.com/asciidoctor/asciidoctor/blob/a1fa9d555eaca835cdc1e30fb2ddc2c83523bd81/lib/asciidoctor/cli/options.rb#L209-L228
